src/app/containers/BrandContainer/index.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import Brand from '../../components/Brand';
+import { ServiceContextConsumer } from '../../components/ServiceContext';
+
+const BrandContainer = () => (
```

Comment:
```
const HeaderContainer = () => (
  <header role="banner">
    <ServiceContextConsumer>
      {({ brandName }) => <Brand brandName={brandName} />}
    </ServiceContextConsumer>
  </header>
 );
```

This would avoid importing the container from the component -- we can get rid of the header component, which is a positive as it currently contains no styling. Also, we can import this into Articles container, keeping a similar level of modularity.

Reply:
I've added a README for the Brand component.